Unable to update to 2.3.0-M02 in maven

Hi,

I am unable to update my dependencies to 2.3.0-M02. 
I receive error:

```
[ERROR] Failed to collect dependencies at org.neo4j.app:neo4j-server:jar:2.3.0-M02 -> org.neo4j:neo4j-ndp-transport-socket:jar:2.3.0-M02: 
Failed to read artifact descriptor for org.neo4j:neo4j-ndp-transport-socket:jar:2.3.0-M02: 
Failure to find org.neo4j:neo4j-data-protocol-parent:pom:2.3.0-M02 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
```

There are no `org.neo4j:neo4j-data-protocol-parent:pom:2.3.0-M02` in maven central.

How this issue can be fixed?

Thanks.

Update.

Build from `community/ndp` leads to:

```
[WARNING] The POM for org.neo4j:neo4j:jar:2.3-SNAPSHOT is missing, no dependency information available
[WARNING] The POM for org.neo4j:neo4j-streams:jar:2.3-SNAPSHOT is missing, no dependency information available
[WARNING] The POM for org.neo4j:neo4j-kernel:jar:tests:2.3-SNAPSHOT is missing, no dependency information available
[WARNING] The POM for org.neo4j:neo4j-io:jar:tests:2.3-SNAPSHOT is missing, no dependency information available
[WARNING] The POM for org.neo4j:neo4j-streams:jar:tests:2.3-SNAPSHOT is missing, no dependency information available
[WARNING] The POM for org.neo4j:neo4j-logging:jar:tests:2.3-SNAPSHOT is missing, no dependency information available
```

Trying `mvn clean install -DminimalBuild -DskipBrowser` from root directory leads to:

```
[ERROR] Failed to execute goal on project neo4j-logging: Could not resolve dependencies for project org.neo4j:neo4j-logging:jar:2.3-SNAPSHOT: Failure to find org.neo4j:neo4j-io:jar:tests:
2.3-SNAPSHOT in http://cifwk-oss.lmera.ericsson.se:8081/nexus/content/groups/public was cached in the local repository, resolution will not be reattempted until the update interval of nex
us has elapsed or updates are forced
```

Trying `mvn clean install -DskipBrowser -DskipTests` from root directory leads to:

```
[ERROR] Failed to execute goal on project neo4j-server: Could not resolve dependencies for project org.neo4j.app:neo4j-server:jar:2.3-SNAPSHOT: Could not find artifact org.neo4j.app:neo4j-browser:jar:2.3-SNAPSHOT in nexus (http://cifwk-oss.lmera.ericsson.se:8081/nexus/content/groups/public) -> [Help 1]
```

---

But, anyway, `mvn clean install -DskipBrowser -DskipTests` is enough to build NDP. Even it fails, because at this point all mandatory artifacts are build.

--- a/community/ndp/kernelextension/pom.xml
+++ b/community/ndp/kernelextension/pom.xml
@@ -6,9 +6,9 @@
 
   <parent>
     <groupId>org.neo4j</groupId>
-    <artifactId>neo4j-data-protocol-parent</artifactId>
+    <artifactId>parent</artifactId>
     <version>2.3-SNAPSHOT</version>
-    <relativePath>../</relativePath>
+    <relativePath>../../..</relativePath>
   </parent>
 
   <properties>
@@ -17,7 +17,6 @@
   </properties>
 
   <artifactId>neo4j-ndp-kernelextension</artifactId>
-  <version>2.3-SNAPSHOT</version>
 
   <packaging>jar</packaging>
   <name>Neo4j - NDP Kernel Extension</name>

--- a/community/ndp/messaging-v1/pom.xml
+++ b/community/ndp/messaging-v1/pom.xml
@@ -6,9 +6,9 @@
 
   <parent>
     <groupId>org.neo4j</groupId>
-    <artifactId>neo4j-data-protocol-parent</artifactId>
+    <artifactId>parent</artifactId>
     <version>2.3-SNAPSHOT</version>
-    <relativePath>../</relativePath>
+    <relativePath>../../..</relativePath>
   </parent>
 
   <properties>
@@ -17,7 +17,6 @@
   </properties>
 
   <artifactId>neo4j-ndp-messaging-v1</artifactId>
-  <version>2.3-SNAPSHOT</version>
 
   <packaging>jar</packaging>
   <name>Neo4j - NDP Messaging Library V1</name>
@@ -69,13 +68,11 @@
     <dependency>
       <groupId>org.asciidoctor</groupId>
       <artifactId>asciidoctorj</artifactId>
-      <version>${ascii.doctor.version}</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.jsoup</groupId>
       <artifactId>jsoup</artifactId>
-      <version>${jsoup.version}</version>
       <scope>test</scope>
      </dependency>
   </dependencies>

--- a/community/ndp/packstream-v1/pom.xml
+++ b/community/ndp/packstream-v1/pom.xml
@@ -4,16 +4,16 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <parent>
     <groupId>org.neo4j</groupId>
-    <artifactId>neo4j-data-protocol-parent</artifactId>
+    <artifactId>parent</artifactId>
     <version>2.3-SNAPSHOT</version>
-    <relativePath>../</relativePath>
+    <relativePath>../../..</relativePath>
   </parent>
+
   <modelVersion>4.0.0</modelVersion>
 
   <name>Neo4j - PackStream</name>
 
   <artifactId>neo4j-ndp-packstream-v1</artifactId>
-  <version>2.3-SNAPSHOT</version>
 
   <properties>
     <license-text.header>GPL-3-header.txt</license-text.header>

--- a/community/ndp/pom.xml
+++ b/community/ndp/pom.xml
@@ -14,12 +14,9 @@
     <license-text.header>GPL-3-header.txt</license-text.header>
     <licensing.prepend.text>notice-gpl-prefix.txt</licensing.prepend.text>
     <componentversion.skip>true</componentversion.skip>
-    <ascii.doctor.version>1.5.2</ascii.doctor.version>
-    <jsoup.version>1.8.1</jsoup.version>
   </properties>
 
   <artifactId>neo4j-data-protocol-parent</artifactId>
-  <version>2.3-SNAPSHOT</version>
 
   <packaging>pom</packaging>
   <name>Neo4j - Data Protocol Project</name>

--- a/community/ndp/runtime/pom.xml
+++ b/community/ndp/runtime/pom.xml
@@ -6,9 +6,9 @@
 
   <parent>
     <groupId>org.neo4j</groupId>
-    <artifactId>neo4j-data-protocol-parent</artifactId>
+    <artifactId>parent</artifactId>
     <version>2.3-SNAPSHOT</version>
-    <relativePath>../</relativePath>
+    <relativePath>../../..</relativePath>
   </parent>
 
   <properties>
@@ -17,7 +17,6 @@
   </properties>
 
   <artifactId>neo4j-ndp-runtime</artifactId>
-  <version>2.3-SNAPSHOT</version>
 
   <packaging>jar</packaging>
   <name>Neo4j - NDP Runtime</name>

--- a/community/ndp/transport-socket/pom.xml
+++ b/community/ndp/transport-socket/pom.xml
@@ -6,9 +6,9 @@
 
   <parent>
     <groupId>org.neo4j</groupId>
-    <artifactId>neo4j-data-protocol-parent</artifactId>
+    <artifactId>parent</artifactId>
     <version>2.3-SNAPSHOT</version>
-    <relativePath>../</relativePath>
+    <relativePath>../../..</relativePath>
   </parent>
 
   <properties>
@@ -17,7 +17,6 @@
   </properties>
 
   <artifactId>neo4j-ndp-transport-socket</artifactId>
-  <version>2.3-SNAPSHOT</version>
 
   <packaging>jar</packaging>
   <name>Neo4j - NDP Socket Transport</name>

--- a/community/ndp/v1-docs/pom.xml
+++ b/community/ndp/v1-docs/pom.xml
@@ -5,9 +5,9 @@
   <modelVersion>4.0.0</modelVersion>
   <parent>
     <groupId>org.neo4j</groupId>
-    <artifactId>neo4j-data-protocol-parent</artifactId>
+    <artifactId>parent</artifactId>
     <version>2.3-SNAPSHOT</version>
-    <relativePath>../</relativePath>
+    <relativePath>../../..</relativePath>
   </parent>
 
   <properties>
@@ -108,19 +108,16 @@
     <dependency>
       <groupId>org.asciidoctor</groupId>
       <artifactId>asciidoctorj</artifactId>
-      <version>${ascii.doctor.version}</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.jruby</groupId>
       <artifactId>jruby-complete</artifactId>
-      <version>1.7.20</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.jsoup</groupId>
       <artifactId>jsoup</artifactId>
-      <version>${jsoup.version}</version>
       <scope>test</scope>
     </dependency>
 

--- a/pom.xml
+++ b/pom.xml
@@ -682,6 +682,24 @@
         <artifactId>jsr311-api</artifactId>
         <version>1.1.2.r612</version>
       </dependency>
+      <dependency>
+        <groupId>org.asciidoctor</groupId>
+        <artifactId>asciidoctorj</artifactId>
+        <version>1.5.2</version>
+        <scope>test</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.jruby</groupId>
+        <artifactId>jruby-complete</artifactId>
+        <version>1.7.20</version>
+        <scope>test</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.jsoup</groupId>
+        <artifactId>jsoup</artifactId>
+        <version>1.8.1</version>
+        <scope>test</scope>
+      </dependency>
     </dependencies>
   </dependencyManagement>
 </project>

